requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 # requirements for local testing with tox, and also for the running test suite
 # or individual tests manually
 tox
+pre-commit
```

Comment:
I have no strong feelings about including this here or not. Given that devs must enable hooks manually, we could also ask them to install pre-commit themselves.